build-info-extractor-docker/src/main/java/org/jfrog/build/extractor/docker/types/DockerImage.java

```diff
@@ -261,14 +267,16 @@ private void findAndSetManifestFromArtifactory(ArtifactoryDependenciesClient dep
         // Try to get manifest, assuming reverse proxy
         String ImagePath = DockerUtils.getImagePath(imageTag);
         ArrayList<String> manifestPathCandidate = new ArrayList<>(DockerUtils.getArtManifestPath(ImagePath, targetRepo, cmdType));
-        logger.info("Trying to fetch manifest from Artifactory.");
+        logger.info("Search manifest for image \""+imageTag+"\" in \""+dependenciesClient.getArtifactoryUrl()+"\" under \"" +targetRepo +  "\" repository");
         int listLen = manifestPathCandidate.size();
         for (int i = 0; i < listLen; i++) {
             try {
-                checkAndSetManifestAndImagePathCandidates(manifestPathCandidate.get(i), dependenciesClient);
+                logger.info("Searching manifest in " + manifestPathCandidate.get(i)+" path.");
```

Comment:
logger.info("Searching manifest in " + manifestPathCandidate.get(i)+" path.");
-->
logger.info("Searching manifest in path: " + manifestPathCandidate.get(i)");